src/sentry/migrations/0259_backfill_integrationfeature.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 2.2.24 on 2021-12-09 23:29
+
+from django.db import migrations
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_target_id(apps, schema_editor):
+    IntegrationFeature = apps.get_model("sentry", "IntegrationFeature")
+
+    for integration_feature in RangeQuerySetWrapperWithProgressBar(
+        IntegrationFeature.objects.all()
+    ):
+        if integration_feature.target_id is None:
+            integration_feature.target_id = integration_feature.sentry_app.id
```

Comment:
Just to be over conservative should we also just set the target_type to make sure we didn't miss anything?

Reply:
Sure - updated. All existing rows will be sentry apps so I just smushed it together.